crates/cli/src/cfg.rs

```diff
@@ -126,16 +135,38 @@ pub enum DALayerOption {
     InMemory,
 }
 
+#[derive(Debug, Default, Clone, Eq, PartialEq, Serialize, Deserialize, ValueEnum)]
+pub enum DBValues {
+    #[default]
+    RocksDB,
+    InMemory,
+    Redis,
+}
+
+#[derive(Args, Deserialize, Clone, Debug)]
+pub struct DatabaseArgs {
+    #[arg(long, value_enum)]
+    db_type: Option<DBValues>,
+
+    /// Path to the RocksDB database, used when `db_type` is `rocks-db`
+    #[arg(long)]
+    rocksdb_path: Option<String>,
```

Comment:
I propose to have the default path `<homePath>/data` instead of `<homePath>`.
Currently, all the data files and config are in the same folder.

Reply:
done